### PR TITLE
:bug: fix attribute-name of multi line tag

### LINKED ISSUE #58: Attributes immediately after newline are not recognised
In this example `attr` has the scope:
- `text.xml`
- `meta.tag.xml`
- `entity.other.attribute-name.localname.xml`
```xml
<tag
     attr="foo">
    bar
</tag>
```

In this example `attr` has the scope:
- `text.xml`
- `meta.tag.xml`
```xml
<tag
attr="foo">
    bar
</tag>
```

### PR DESCRIPTION
**Pull request by ayapi**
this pr will fix #58 . please review

### CODE REVIEW

File: grammars/xml.cson [hunk 1]
```diff
@@ -327,7 +327,7 @@
             'name': 'punctuation.separator.namespace.xml'
           '4':
             'name': 'entity.other.attribute-name.localname.xml'
-        'match': ' (?:([-_a-zA-Z0-9]+)((:)))?([-_a-zA-Z0-9]+)='
+        'match': '\\s*(?:([-_a-zA-Z0-9]+)((:)))?([-_a-zA-Z0-9]+)='
```
Comment (50Wliu) [inline grammars/xml.cson]: Should be \\s+ I think.

Comment (50Wliu) [inline grammars/xml.cson]: Oh wait, never mind, that doesn't fix the reported bug.  Hmm.

Comment (ayapi) [inline grammars/xml.cson]: wow! sorry, i made a mistake! please wait...

**Comment by ayapi** [comment]
@50Wliu sorry about earlier!
i've pushed revised commit just now, please check

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**